crossing-training.lic

```diff
@@ -413,13 +413,22 @@ class CrossingTraining
   end
 
   def research
+    if @settings.cyclic_research
```

Comment:
Is this boolean necessary, or is the presence of `crossing_cyclic_research_spells` (or `cyclic_research_skills`) enough to indicate this should be done?